openslides/utils/auth.py

```diff
@@ -39,6 +39,41 @@ def has_perm(user: Optional[CollectionElement], perm: str) -> bool:
     return has_perm
 
 
+def in_some_groups(user: Optional[CollectionElement], groups: Union[List[int], List[Group]]) -> bool:
+    """
+    Checks that user is in at least one given group. Groups can be given as a list
+    of ids or group instances.
+
+    User can be a CollectionElement of a user or None.
+    """
+
+    if len(groups) > 0:
+        if isinstance(groups[0], Group):
+            groups = [group.id for group in groups]  # type: ignore
+    else:
+        return False  # early end here, if no groups are given.
+
+    # Convert user to right type
+    # TODO: Remove this and make use, that user has always the right type
+    user = user_to_collection_user(user)
+    if user is None and not anonymous_is_enabled():
+        in_some_groups = False
+    elif user is None:
+        # Use the permissions from the default group with id 1.
+        in_some_groups = 1 in groups
```

Comment:
Hm. This looks dangerous because in future we may change the meaning of Group pk 1 (superadmin group). Proposal: Let's introduce a upper case variable for this.

Reply:
Where should i create it? in `auth.py` or somewhere in the users?

Reply:
I am not sure. Maybe somewhere in users.

Reply:
According to #3859 I'll add it to auth.